foia_hub/context_processors/google_analytics.py

```diff
@@ -0,0 +1,6 @@
+from django.conf import settings
+
+def google_analytics(request):
+    """ This should make the Google ANALYTICS_ID available to all templates.
+    """
+    return {'ANALYTICS_ID': settings.ANALYTICS_ID}
```

Comment:
why not use this as a constant, e.g. https://github.com/18F/peacecorps-site/blob/master/peacecorps/peacecorps/settings/production.py#L33